packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -675,8 +675,13 @@ describe('Collection aggregations tab', function () {
       await errorDetailsJson.waitForDisplayed();
 
       // exit details
+      // leafygreen autofocus triggers a tooltip on the error code element,
+      // "Tab" to remove the focus
+      await browser.keys('Tab');
```

Comment:
All three e2e tests failing after the update were caused by the leafygreen autofocus first thing in the modal behavior. The way of dealing with it is the same everywhere and it's more than two places, but I still didn't feel like it warrants a shared helper of some kind. Tell me if you have a different opinion on this